### DISCUSSION TITLE
Changing the subnet values of captain-overlay-network

### DISCUSSION THREAD

**Discussion by omcdevadmin** [caprover/caprover#995] [Q&A] [answered]
Hi!

Looks like it's not possible to host more than 200 containers inside a caprover installation (including several IPs reserved for swarm and docker), is there a safeway to change this value and keep it persistent : 

"Subnet": "10.0.1.0/24",

I'd like something like 10.0.0.0/16 which will bring 65,536 IPs. 

Not that it should be a good idea to host more than 200 containers on a two or three nodes docker swarm installation, including replicated nodes and everything 256 IPs seems a little low.

Thanks!

**Comment by maietta** [comment, marked as answer]
This would not be a function of Caprover but instead the Docker networking. Docker has undergone many changes since this question as posted on Stackoverflow with a possible solution provided: https://stackoverflow.com/questions/44003663/changing-default-subnet-for-docker-custom-networks

Another article discusses this issue as well: https://support.skyformation.com/hc/en-us/articles/360009195759-How-To-Change-the-Docker-IP-address-space

Also found this but this pertains only to the networks created, not specifically the default subnet: https://docs.docker.com/engine/reference/commandline/network_create/

**Reply by maietta** [reply]
To be more clear about this, I would say that your address space provided by the docker daemon is limited because it's a /24 space. You can change the behavior of Docker itself but I would definitely test this on a throw-away VPS or something first.

**Reply by omcdevadmin** [reply]
Thanks! I will do some research around that and definitely try first on a cheap VPS !
